zcncore/wallet_base.go

```diff
@@ -1009,6 +1013,42 @@ func GetStakePoolUserInfo(clientID string, cb GetInfoCallback) (err error) {
 	return
 }
 
+// GetUserLockedTotal get total token user locked
+// # Inputs
+//   - clientID wallet id
+func GetUserLockedTotal(clientID string) (int64, error) {
+
+	err := checkSdkInit()
+	if err != nil {
+		return 0, err
+	}
+
+	var url = withParams(STORAGESC_GET_USER_LOCKED_TOTAL, Params{
+		"client_id": clientID,
+	})
+	cb := createGetInfoCallback()
+	go GetInfoFromSharders(url, OpStorageSCGetStakePoolInfo, cb)
+	info, err := cb.Wait()
+	if err != nil {
+		return 0, err
+	}
+
+	result := make(map[string]int64)
+
+	err = json.Unmarshal([]byte(info), &result)
+	if err != nil {
+		return 0, errors.Throw(err, "invalid json format")
+	}
+
+	total, ok := result["total"]
+	if ok {
+		return total, nil
+	}
+
+	return 0, errors.New("", "invalid result")
```

Comment:
could be simple standard error `errors.New("invalid request")

Reply:
@peterlimg  fixed